<ui version="4.0" >
 <class>WidgetCreateGasParticles</class>
 <widget class="QWidget" name="WidgetCreateGasParticles" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QGridLayout" >
      <property name="verticalSpacing" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Region volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lineEditVolume" >
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Particles count:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="lineEditCount" >
        <property name="text" >
         <string>20</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Concentration:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="lineEditConcentration" >
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Particles mass:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="lineEditMass" >
        <property name="text" >
         <string>1</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Temperature:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="lineEditTemperature" >
        <property name="text" >
         <string>1e21</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Mean velocity:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLineEdit" name="lineEditMeanVelocity" >
        <property name="text" >
         <string>(0,0)</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditVolume</tabstop>
  <tabstop>lineEditCount</tabstop>
  <tabstop>lineEditConcentration</tabstop>
  <tabstop>lineEditMass</tabstop>
  <tabstop>lineEditTemperature</tabstop>
  <tabstop>lineEditMeanVelocity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
